website/app/templates/support.hbs

```diff
@@ -1,8 +1,20 @@
 {{page-title 'Support'}}
 
 <Doc::Page::Stage>
+  <Doc::Page::Cover @title="Support" />
   <Doc::Page::Content>
-    <h1>Support</h1>
-    <p>TBD what this page will contain</p>
+    <p class="doc-text-body">We are happy to provide support to HashiCorp product teams that use our design system.Here are some of the ways to get help when you get stuck.</p>
```

Comment:
Missing a space in between the sentences, also I would say "if" they get stuck vs. "when" to sound more positive:

"We are happy to provide support to HashiCorp product teams that use our design system.Here are some of the ways to get help when you get stuck."
=>
"We are happy to provide support to HashiCorp product teams that use our design system. Here are some of the ways to get help if you get stuck."